Data/models/Device.swift

```diff
@@ -110,4 +110,13 @@ class Device: NSManagedObject, Syncable {
         }
     }
     
+    // Needed for testing only. No need to set it public, @testable annotation allows use of internal methods.
+    class func clearSharedDevice() {
```

Comment:
Same, why does this need to be here instead of in the unit test target?

Reply:
Same goes here, I need to drop `private` access for `shared`